### PR TITLE
Issue#8

### PR DESCRIPTION
**Pull request by KylianLM**
@ChibiFR #8

### CODE REVIEW

File: src/Helpers/Crypto.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+namespace Martine\Helpers;
+
+/**
+ * Class Crypto
+ * @package Martine\Helpers
+ */
+class Crypto
+{
+    /**
+     * @var string
```
Comment (ChibiFR) [inline src/Helpers/Crypto.php]: Add description: "Random key of the Martine application"

File: src/Helpers/Crypto.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+     */
+    private $appKey;
+
+    /**
+     * Crypto constructor.
```
Comment (ChibiFR) [inline src/Helpers/Crypto.php]: Add new line between description and annotations

File: src/Helpers/Crypto.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+     * @param string $appKey
+     */
+    public function __construct(string $appKey)
+    {
+        $this->appKey = $appKey;
+    }
+
+    /**
+     * Crypt a string with a appKey
```
Comment (ChibiFR) [inline src/Helpers/Crypto.php]: $appKey, not appKey

File: src/Helpers/Crypto.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+     * @param string $str
+     * @return string
+     */
+    public function encrypt(string $str): string
+    {
+        return crypt($str, $this->appKey);
+    }
+
+    /**
+     * Checks if a given string matches a given crypt
```
Comment (ChibiFR) [inline src/Helpers/Crypto.php]: Check, not checks. No 3d person use in doc

File: src/Helpers/Crypto.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+     *
+     * @param string $knowStr
```
Comment (ChibiFR) [inline src/Helpers/Crypto.php]: $knownStr instead of $knowStr

**Comment by KylianLM** [comment]
@ChibiFR 
done

**Pull request merged**